kong/db/dao/plugins.lua

```diff
@@ -277,6 +282,10 @@ function Plugins:load_plugin_schemas(plugin_set)
         handler = handler()
       end
 
+      if handler._go then
+        go_plugins_cnt = go_plugins_cnt + 1
```

Comment:
Can this be calculated above in the `load_plugin_handler` function.

Reply:
`load_plugin_handler` doesn't have a view of all plugins - it's called for each plugin - so `load_plugin_schemas` is the place where this sum can be done

Reply:
Cool, thanks for clarifying!